### DISCUSSION TITLE
Add settings for line blame for all the lines

### DISCUSSION THREAD

**Discussion by tomdelahaba** [gitkraken/vscode-gitlens#2068] [Ideas]
I really like the line blame at the end of the lines. Its not disruptive and usually I find this information pretty important.

I would like to have a option (settings) where I should turn this on for all lines (not just the active one) so I do not need to click on the line, wait few milliseconds till the line blame appears, etc.

**Comment by eamodio** [comment]
Does the File Blame work for your use-case?
